app/models/topic.rb

```diff
@@ -54,9 +59,9 @@ def self.recent_topics
                        ON forums.id = topics.forum_id
                LEFT OUTER JOIN forumers
                             ON forumers.forum_id = forums.id
-        WHERE forumers.id IS NULL
+        WHERE forumers.id IS NULL #{constraint ||= ""} 
```

Comment:
Useless assignment to variable - `constraint`.<br>Prefer single-quoted strings inside interpolations.<br>Trailing whitespace detected.